README.md

```diff
@@ -10,7 +10,16 @@ Crystal is a programming language with the following goals:
 * Have compile-time evaluation and generation of code, to avoid boilerplate code.
 * Compile to efficient native code.
 
-## Build
+## Contributing to These Docs
```

Comment:
I agree.

"Contributing" would be confusing because a user might think that it's talking about contributing to the language.

"Contributing to Documentation" sounds nice, but doesn't really specify that there are API docs that are important as well.

Any ideas?